lib/core/src/lib/dialogs/dialog/dialog.component.ts

```diff
@@ -68,11 +68,11 @@ export class DialogComponent implements OnDestroy {
             });
 
             if (data.isConfirmButtonDisabled$) {
-                data.isConfirmButtonDisabled$.pipe(takeUntil(this.onDestroy$)).subscribe((value) => this.isConfirmButtonDisabled$.next(value));
+                data.isConfirmButtonDisabled$.pipe(takeUntilDestroyed(this.destroyRef)).subscribe((value) => this.isConfirmButtonDisabled$.next(value));
```

Comment:
We can use takeUntilDestroy without destroyRef here probably because it's inside constructor